wall.opam

```diff
@@ -8,10 +8,10 @@ bug-reports: "https://github.com/def-lkb/wall"
 license: "BSD3"
 dev-repo: "https://github.com/def-lkb/wall.git"
 build: [
-  [make]
+  [jbuilder build]
 ]
 install: [make "install"]
-remove: ["ocamlfind" "remove" "wall"]
+remove: ["opam-installer"]
```

Comment:
AFAIK with jbuilder we should use `install:` and `remove:` since everything is in the `wall.install` file, just

```
build: [
  ["jbuilder" "build" "--only" "wall" "--root" "." "-j" jobs "@install"]
]
```